man/pam_systemd_home.xml

```diff
@@ -51,8 +51,31 @@
         coming back from suspend. It is recommended to set this parameter for all PAM applications that have
         support for automatically re-authenticating via PAM on system resume. If multiple sessions of the
         same user are open in parallel the user's home directory will be left unsuspended on system suspend
-        as long as at least one of the sessions does not set this parameter. Defaults to
-        off.</para></listitem>
+        as long as at least one of the sessions does not set this parameter to on. Defaults to
+        off.</para>
+
+        <para>Note that TTY logins generally do not support re-authentication on system resume, there's
+        simply no concept in TTY sessions for that. Re-authentication on system resume is hence primarily a
+        concept implementable in graphical environments, in the form of lock screens brought up automatically
+        when the system goes to sleep. Note that this means that if a user concurrently uses graphical login
```

Comment:
I'd shorten this a bit:
        Note that TTY logins generally do not support re-authentication on system resume.
       Re-authentication on system resume is primarily a
        concept implementable in graphical environments, in the form of lock screens brought up automatically
        when the system goes to sleep. This means that